nyaa/templates/view.html

```diff
@@ -159,10 +159,10 @@ <h3 class="panel-title">
 					<small data-timestamp-swap data-timestamp-title data-timestamp="{{ comment.edited_utc_timestamp }}" title="{{ comment.edited_time }}">(edited)</small>
 					{% endif %}
 					<div class="comment-actions">
-						{% if g.user.id == comment.user_id and not comment.editing_limit_exceeded %}
+						{% if g.user.id == comment.user_id and not comment.editing_limit_exceeded and (not torrent.locked or g.user.is_moderator) %}
```

Comment:
"... and (not torrent.locked or comment_form)"
if they can comment they should be able to edit, easier for maintenance.